ephemeris/run_data_managers.py

```diff
@@ -40,6 +50,75 @@ def wait(gi, job):
         time.sleep(30)
 
 
+def data_table_entry_exists(tool_data_client, data_table_name, entry, column='value'):
+    '''Checks whether an entry exists in the a specified column in the data_table.'''
+    try:
+        data_table_content = tool_data_client.show_data_table(data_table_name)
+    except:
```

Comment:
it's always preferable to use `except Exception`, or even better the exact exception class that you are expecting in this case, so that when troubleshooting we are not mislead by `'Table "%s" does not exist'`. A bunch of reasons for this is listed in this thread https://stackoverflow.com/questions/14797375/should-i-always-specify-an-exception-type-in-except-statements